src/WebJobs.Script.WebHost/FileMonitoringService.cs

```diff
@@ -73,9 +74,13 @@ private void InitializeFileWatchers()
         {
             _debugModeFileWatcher = new AutoRecoveringFileSystemWatcher(_hostLogPath, ScriptConstants.DebugSentinelFileName,
                    includeSubdirectories: false, changeTypes: WatcherChangeTypes.Created | WatcherChangeTypes.Changed);
-
             _debugModeFileWatcher.Changed += OnDebugModeFileChanged;
-            _logger.LogTrace("Debug file watch initialized.");
+            _logger.LogInformation("Debug file watch initialized.");
```

Comment:
I'd say these kinds of logs should be `Debug` -- then they'll show up in the system logs but not by default in console/file/appinsights logs. They're mostly useful to us but won't mean much to the general users. This applies to all the ones that went from `Trace` -> `Information`. I'd vote to move them to `Debug`.

Reply:
Agreed. Changing them to Debug